pkg/client/client.go

```diff
@@ -178,6 +179,10 @@ func NewClientFromKeys(accessKey, secretAccessKey, region string) (Client, error
 	if err != nil {
 		return nil, err
 	}
+	s.Handlers.Build.PushBackNamed(request.NamedHandler{
+		Name: "sigs.k8s.io.ClusterAPIProviderAWS",
```

Comment:
Is it allowed to use `/` and `-` in the name?  `openshift.io/cluster-api-provider-aws` maybe?

Reply:
done with https://github.com/openshift/cluster-api-provider-aws/compare/815c929f51df180fcf72af0a0feb6156a893fb6f..8338c4fb0b0ad3b2e62af0fb203173ca4bd36f39